src/DFG.cpp

```diff
@@ -49,6 +38,75 @@ DFG::DFG(Function& t_F, list<Loop*>* t_loops, bool t_targetFunction,
 
 }
 
+// Specilized fusion for the nonlinear operations.
+void DFG::nonlinear_combine() {
+  tuneForBitcast();
+  combineMulAdd(2);
```

Comment:
The argument `(1)` within the func needs to be exclusive to others? Then why `combinePhiAdd()` and `combineAddAdd()` have the same argument of `1`?

Reply:
I refactor the codes for fusion, details will be illustrated in the latest comment.